package.json

```diff
@@ -23,6 +23,7 @@
     "grunt-contrib-watch": "^1.1.0",
     "grunt-eslint": "^22.0.0",
     "grunt-webpack": "^3.1.3",
+    "jquery": "^3.5.1",
```

Comment:
Note for future me:
If jquery is added here, maybe the fetch in `index.html` isn't needed anymore.